api/src/test/kotlin/edu/wgu/osmt/collection/CollectionControllerTest.kt

```diff
@@ -31,6 +39,8 @@ internal class CollectionControllerTest @Autowired constructor(
     @Autowired
     lateinit var collectionController: CollectionController
 
+    var authentication: Authentication = Mockito.mock(Authentication::class.java)
```

Comment:
In Kotlin, we often use mockk. I can't give you a deep reason why, but there are times/places where Mockito is problematic. I'm not asking for a change here, just please be aware.